src/pudl/extract/excel.py

```diff
@@ -206,6 +207,13 @@ def extract(self, years):
                 page)).difference(df.columns)
             empty_cols = pd.DataFrame(columns=missing_cols)
             df = pd.concat([df, empty_cols], sort=True)
+            if len(self.METADATA._column_map[page].index) != len(df.columns):
+                # raise AssertionError(
+                logger.info(
+                    f'Columns for {page} are off: should be '
+                    f'{len(self.METADATA._column_map[page].index)} but got '
+                    f'{len(df.columns)}'
+                )
```

Comment:
I felt it as a logger for the moment a) because it wasn't working with the other eia datasets. I finally got it to work so that should be okay. but also b) bc if it straight up failed I wasn't able to actually look at the outputs. I'd like to make this a warning, so it screams at you very clearly, but keeps moving.